homeassistant/components/roborock/device.py

```diff
@@ -49,9 +50,15 @@ async def send(
             response: dict = await self._api.send_command(command, params)
         except RoborockException as err:
             raise HomeAssistantError(
-                f"Error while calling {command.name if isinstance(command, RoborockCommand) else command} with {params}"
+                f"Error while calling {command}.",
+                translation_domain=DOMAIN,
+                translation_key="command_failed",
+                translation_placeholders={
+                    "command": command.name
+                    if isinstance(command, RoborockCommand)
+                    else command,
```

Comment:
Let's avoid multi-line ternaries in our codebase.

Reply:
pulled out the ternary operator